docker-compose.yml

```diff
@@ -10,14 +10,7 @@ services:
             - ./docs:/srv/jekyll
         ports:
             - "4000:4000"
-    web:
```

Comment:
I took a shot at updating the pull request in order to remove docker-compose.yml via the instructions in that link, let me know if it didn't work.